posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,77 @@
+
+Bear in mind that changing this configuration option will also affect the behaviour of the Project Explorer pane.
+
+## Open files externally
+
+Additionally, we added a context menu action called Open externally to open files with the Operating System default program associated with the file type. To use this action, go to the file you want to open in the files tab and right-click it to see the context menu. Then select “Open Externally” and the file will be opened outside Spyder, with the default program associated with its file type, depending on the Operating System.
+
+![Open Externally](/images/spyder-files-open-externally.png)
+
+Moreover, now it is possible to select several files to perform a specific action thanks to the multi-select functionality added. For this, press CTRL or SHIFT and Click on the files that are going to be selected without releasing the CTRL/SHIFT. It is also possible to select the files without doing it one by one, but selecting a file with the CTRL/SHIFT clicked and selecting a not contiguous file to this one which will select all the files between them.
+By doing this, it is possible to execute some of the actions available in the context menu for all the files selected including delete, rename, move, copy and open externally.
+
+
+![Select Files](/images/spyder-files-select-files.png)
+
+## Absolute and relative path handling
+
+Another functionality added was the ability to copy and paste files and their absolute or relative paths. To access these actions, go to the context menu by right-clicking a file in the files tab. 
+
+![Copy Path](/images/spyder-files-copy-path.png)
+
+With this, a file can be copied from the files tab and pasted anywhere else directly from Spyder. The “Copy Absolute Path” and “Copy Relative Path” actions, give us access to the files’ path, to be pasted as text. 
+
+This way, when pasting the absolute path of a file, we get the complete path of the file starting from the root.
+
+![Absolute Path](/images/spyder-files-absolute-path.png)
+
+When using the option “Copy Relative Path”, we get the path of the file relative to the directory in which we are located.
+
+![Relative Path](/images/spyder-files-relative-path.png)
+
+Finally, files are now displayed along with icons depending on their file type. There are different icons for each programming language file extension, including c, cpp, csharp, java, python, r, swift, for each different file type including jpg, mp3, m4a and other file extensions including txt and tex. This allows the users to visually identify what type of file they are seeing in the files pane, in order to select the correct associated applications to open these files.
+
+![File Extensions](/images/spyder-files-file-extensions.png)
+
+With this new features added to Spyder's file management behavior and UI design in the files plugin, we think we achieved to improve the experience of the user managing files in Spyder.
```

Comment:
```suggestion
With these new features and UI options in Spyder's `Files` pane, we hope you will enjoy the improved file management experience coming in Spyder 4! We would like to thank Quansight, our donors, and most of all our dedicated users and contributors from around the world for making these features possible. The final version will be released very soon, but if you'd like to try it out now and share your feedback, follow the instructions on our [Github](https://github.com/spyder-ide/spyder/releases). Happy Spydering!
```

* Fix many grammar and syntax errors
* Avoid repetition, awkwardness and redundancy
* Use standard terminology
* Use correct names from the UI text
* Add thank you to Quansight, donors and users
* Add call to action for users to try out these new features, and put them into context of when they land in Spyder